iron-spi-aws/src/main/java/io/axway/iron/spi/aws/AwsUtils.java

```diff
@@ -36,29 +45,54 @@ public static void setAws(AwsClientBuilder builder, //
      * @param action the action to retry
      * @param retryLimit retry number limit
      * @param durationInMillis duration between each retry
-     * @throws LimitExceededException after retry exhausted
+     * @throws AwsException after retry exhausted
      */
     public static <T> T performAmazonActionWithRetry(String actionLabel, Supplier<T> action, int retryLimit, int durationInMillis) {
```

Comment:
look likes than retryLimit/durationInMillis are rather a configuration of a (missing) object than parameters call (see usages)